baseapp-chats/baseapp_chats/graphql/mutations.py

```diff
@@ -168,6 +168,16 @@ def mutate_and_get_payload(cls, root, info, profile_id, participants, is_group,
         room.participants_count = len(created_participants)
         room.save()
 
+        if is_group:
+            send_message(
+                message_type=Message.MessageType.SYSTEM_GENERATED,
+                room=room,
+                profile=None,
+                user=None,
+                content='{content_linked_profile} created group "' + title + '"',
```

Comment:
@tsl-ps2 @nossila can we generate the parsing on the backend instead of parsing it on the frontend? I _think_ we would have everything we need (like access to the current profile via `info.context.user.current_profile`), don't we?